tests/test_association_task.py

```diff
@@ -192,24 +285,33 @@ def _run_association_and_retrieve_objects(self, create_objects=False):
             start_id=10,
             scatter_arcsec=-1)
         for dia_source in dia_sources:
-            dia_source["psFlux"] = 10000.
-            dia_source["psFluxErr"] = 100.
+            dia_source['ccdVisitId'] = \
+                self.exposure.getInfo().getVisitInfo().getExposureId()
+            dia_source["midPointTai"] = \
+                self.exposure.getInfo().getVisitInfo().getDate().get(system=dafBase.DateTime.MJD)
+            dia_source["psFlux"] = 10000 / self.flux0
+            dia_source["psFluxErr"] = \
+                np.sqrt((100 / self.flux0) ** 2 + (10000 * self.flux0_err / self.flux0 ** 2) ** 2)
+            dia_source["apFlux"] = 10000 / self.flux0
+            dia_source["apFluxErr"] = \
+                np.sqrt((100 / self.flux0) ** 2 + (10000 * self.flux0_err / self.flux0 ** 2) ** 2)
+            dia_source["filterName"] = self.exposure.getFilter().getName()
+            dia_source["filterId"] = self.exposure.getFilter().getId()
+            dia_source["x"] = 0
+            dia_source["y"] = 0
+            dia_source["snr"] = 10
```

Comment:
Are these all mandatory columns? :fearful: I'm a bit worried about code duplication with the code you're supposed to be testing...

Reply:
"x" and "y" are non-Nullable in the Ppdb thus I set them here. The association step takes place in RA/DEC and these columns are not touched.